js/demo-app/src/index.js

```diff
@@ -62,18 +62,18 @@ document.addEventListener("DOMContentLoaded", () => {
   window.onhashchange = function() {
     if (routing) {
       // On hash change & routing mode, update route attribute
-      window.setRoute(window.location.hash);
+      window.setRoute(window.location.hash.slice(1));
     }
   };
 
   document
     .getElementById("router")
     .addEventListener("navRequest", function(data) {
       if (window.routing) {
-        location.hash = data.detail.hash;
+        location.hash = data.detail.fragment;
         // On navRequest & routing mode, change url
       }
-      window.setRoute(data.detail.hash);
+      window.setRoute(data.detail.fragment);
```

Comment:
The change from `hash` to `fragment` is the result of switching to the Elm 0.19 [Url package](https://package.elm-lang.org/packages/elm/url/latest/Url) for Urls.